packages/runtime/container-runtime/src/summaryManager.ts

```diff
@@ -296,19 +295,14 @@ export class SummaryManager extends TypedEventEmitter<ISummaryManagerEvents> imp
     private async delayBeforeCreatingSummarizer(): Promise<boolean> {
         // throttle creation of new summarizer containers to prevent spamming the server with websocket connections
         let delayMs = this.startThrottler.getDelay();
-        if (delayMs > 0 && delayMs > this.startThrottler.maxDelayMs) {
-            this.emit(
-                "summarizerWarning",
-                createSummarizingWarning("summaryManagerCreateSummarizerMaxThrottleDelay", false),
-            );
-        }
 
         // We have been elected the summarizer. Some day we may be able to summarize with a live document but for
         // now we play it safe and launch a second copy.
         this.logger.sendTelemetryEvent({
             eventName: "CreatingSummarizer",
             throttlerDelay: delayMs,
             initialDelay: this.initialDelayMs,
+            delayMaxStartThrottler: delayMs > this.startThrottler.maxDelayMs,
```

Comment:
Not sure what "throttler" means in this name (oh... I see, `startThrottler`).  Maybe something like "delayExceedsMax", or another idea is to log the max instead of the comparison boolean result.  Like `startThrottlerMaxDelayMs: this.startThrottler.maxDelayMs`

Reply:
I like the idea of just keeping `startThrottlerMaxDelayMs` instead of the comparison 😄